openwisp_notifications/migrations/0003_notificationsetting.py

```diff
@@ -11,14 +11,20 @@
 from openwisp_notifications.types import NOTIFICATION_CHOICES
 
 
+def get_swapped_model(apps, app_name, model_name):
+    model_path = swapper.get_model_name(app_name, model_name)
+    app, model = swapper.split(model_path)
+    return apps.get_model(app, model)
+
```

Comment:
Let's move it to `migrations/__init__.py`, since if might be required and used in other files too.